### ISSUE TITLE
Parameters for Separated-Atom and United-Atom Limits

### ISSUE THREAD

**Issue by PaulWAyers** [theochem/AtomDB#6]
It's useful to have key quantities related to the separated and united atom limits. These include dispersion coefficients (C6, C8, C10, etc.) for homonuclear diatomics and (C9) triatomics. Data isn't available universally, but the Grimme D2 parameters for C6 are available in Psi4.
    https://github.com/psi4/psi4/blob/2cd33eda01b7018a23739d00c1cdd51ca87faa64/psi4/src/psi4/libdisp/dispersion_defines.h#L227

Beyond these values, I can find data for several diatomics, and good benchmark data for hydrogen, but limited "universal" data. Over time, we should try to put benchmark C6, C8, C10, and C9 coefficients into the database where possible.

@gabrielasd if you can set up the fields for these quantities, then @SakshiTak can add the data as she accumulates it.

For the united-atom limit, it's helpful to have the electron density at the nucleus provided as a parameter.

**Comment by gabrielasd** [comment]
@PaulWAyers is it OK to make these fields "global" like the cov_radii, vdw_radii and mass? Meaning that they are always populated no matter the origin of the dataset (HF results or NIST data). 
They would be default values in place of specific ones if they can be computed for a given thataset.

**Comment by PaulWAyers** [comment]
I think these fields can be global, yes.

@gabrielasd do we have the atomic energies from NIST uploaded? I didn't see them but I may be looking the wrong place. I saw a few examples but not the whole database.

**Comment by gabrielasd** [comment]
@PaulWAyers , yes, the atomic energies are included for the NIST dataset (only the ground state one). 
In the compilation script [here](https://github.com/theochem/AtomDB/blob/dev/atomdb/datasets/nist/__init__.py), the lowest energy is taken in line [84](https://github.com/theochem/AtomDB/blob/c9b310a2d2b6eeeda37861e56093bfe430885d39/atomdb/datasets/nist/__init__.py#L84) for the neutral and cations and line [101](https://github.com/theochem/AtomDB/blob/c9b310a2d2b6eeeda37861e56093bfe430885d39/atomdb/datasets/nist/__init__.py#L101) for anions. And given to the Species class in line 142.

The example in [Getting_Started](https://github.com/theochem/AtomDB/blob/dev/examples/notebooks/Getting_Started.ipynb) notebook is not for the NIST dataser, but under, Getting Atomic Properties/Scalars section, the way to retrieve the property from an `atom` instance of Species class is shown as:
`print("Energy (Hartree):", atom.energy)`

However as you say, the database is not fully compiled. The only complete database uploaded to the repo is the Slater one. For the NIST, numeric and gaussian sets, I was considering compiling the neutral, +1 and -1 species an placing them in graham.

**Comment by PaulWAyers** [comment]
@gabrielasd do we have the raw files/data for the NIST dataset? I think @tczorro had compiled them for all stable charges and multiplicities of the elements. That would probably be enough for our imminent applications.

**Comment by tczorro** [comment]
@PaulWAyers I have the raw txt format (one for each atom with specific charge) and h5 format database. The h5 should be the same one Michelle or Farnaz have worked on

**Comment by gabrielasd** [comment]
@PaulWAyers, yes, we have the h5 format file that @tczorro is referring to.
It is sitting in the `oldmaster` branch: https://github.com/theochem/AtomDB/tree/oldmaster/atomdb/data, a file named: database_beta_1.3.0.h5


The compilation script for the NIST dataset in the current branches of the repo (master and dev) is this one:

https://github.com/theochem/AtomDB/blob/dev/atomdb/datasets/nist/__init__.py#L16

where the raw file gets called in lines 78 and 95 to get the energy of the ground state for a given species (element,charge,multiplicity): https://github.com/theochem/AtomDB/blob/5369631832658ff880d24b04de63500deb0023d1/atomdb/datasets/nist/__init__.py#L78

**Comment by PaulWAyers** [comment]
It's also useful to have the mass of the most abundant isotope for each element. (I seem to recall having that in an issue but I can't find it right now.)

I think @marco-2023 has this data already in `gopt` based on the NIST data. 

https://physics.nist.gov/cgi-bin/Compositions/stand_alone.pl

**Comment by gabrielasd** [comment]
@PaulWAyers 
The mass of the most abundant isotopes is being parsed from a csv file [here](https://github.com/theochem/AtomDB/blob/85031c5fc08b7c920a093c84ab4f6458b5d48825/atomdb/api.py#L527) 
That csv also includes van der Waals and covalent radii from different sources.

I'll take the data from Marcos, as it is updated, and combine it with what we have.

**Comment by PaulWAyers** [comment]
Thanks. I think we want atomic masses (isotopic average) and most-abundant masses. With those two pieces of information, we're in great shape.
